tool/util.py

```diff
@@ -0,0 +1,79 @@
+# Lint as: python3
```

Comment:
Since this is the main file, could you give the file a more descriptive name? pv_tree_generator or something like that?

Reply:
This is not the main file, I corrected the .sh file and script to avoid confusion